src/components/SpeakerCard/index.view.tsx

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+import * as React from "react";
+import cx from "classnames";
+import "./SpeakerCard.scss";
+
+interface SpeakerCardProps {
+  speakerName: string;
+  company: string;
+  companyLink?: string;
+  linkedin?: string;
+  isOpening?: boolean;
+}
+
+const SpeakerCard: React.FC<SpeakerCardProps> = ({
+  speakerName,
+  company,
+  companyLink,
+  linkedin,
+  isOpening,
+}: SpeakerCardProps) => {
+  const extension = speakerName === "Coming Soon" ? "svg" : "jpeg";
+  return (
+    <div className="SpeakerCard">
+      <div className="SpeakerCard__firstLayer">
+        <div className="SpeakerCard__secondLayer">
+          <div className="SpeakerCard__thirdLayer">
+            <div className="SpeakerCard__imageContainer">
```

Comment:
Would it actually be possible to combine `__thirdLayer` and `__imageContainer`? It looks like they apply to the same visual component, so I'm not sure why we need something so fine-grained.

What would be even better is if we could consolidate `__thirdLayer`, `__imageContainer`, *and* `__image` because the the parent container of the image just looks like padding we can apply, thus greatly reducing the verbosity of our HTML structure.

Reply:
I wouldn't be opposed to combining these two layers; however, the image becomes distorted if we have it take up the entire width of the card.